### PR TITLE
Dedent over-indented test.

### PR DESCRIPTION
**Pull request by Lukasa**
Spotted this while I was working on #497.

### CODE REVIEW

File: test/test_basic_logic.py [hunk 1]
```diff
@@ -1699,21 +1699,21 @@ def sanity_check_data_frame(data_frame,
...
-             expected_flow_controlled_length)
-        )
-        if expect_padded_flag:
-            assert 'PADDED' in data_frame.flags
-        else:
-            assert 'PADDED' not in data_frame.flags
+    assert(
+        (data_frame.flow_controlled_length ==
+            expected_flow_controlled_length)
+    )
```
Comment (alexwlchan) [inline test/test_basic_logic.py]: With the extra dedenting, this assertion fits on a single line.

**Comment by Lukasa** [comment]
Thanks @alexwlchan, I've addressed that.

File: test/test_basic_logic.py [hunk 2]
```diff
@@ -1699,21 +1699,20 @@ def sanity_check_data_frame(data_frame,
...
-            (data_frame.flow_controlled_length ==
-             expected_flow_controlled_length)
-        )
-        if expect_padded_flag:
-            assert 'PADDED' in data_frame.flags
-        else:
-            assert 'PADDED' not in data_frame.flags
+    assert (
+        data_frame.flow_controlled_length == expected_flow_controlled_length
+    )
```
Comment (alexwlchan) [inline test/test_basic_logic.py]: Sorry, I should have been clearer. You can pull it onto a single line without the parentheses, i.e.,

```python
    assert data_frame.flow_controlled_length == expected_flow_controlled_length
```

Comment (Lukasa) [inline test/test_basic_logic.py]: Ah...yes. I thought I tried that and it didn't work, but clearly I'm an idiot. That's what happens when you're sick for a week. :(

**Pull request merged**